### PR TITLE
Create ruleset: Refunite.org

### PR DESCRIPTION
**Pull request by jtracey**

### CODE REVIEW

**Comment by jtracey** [comment]
error looks spurious to me

File: src/chrome/content/rules/Refunite.org.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!--
+	Nonfunctional hosts in *.refunite.org:
+
+	- www: mismatch
```
Comment (Bisaloo) [inline src/chrome/content/rules/Refunite.org.xml]: blog.refunite.org

File: src/chrome/content/rules/Refunite.org.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+-->
+<ruleset name="Refunite.org">
+	<target host="refunite.org" />
+	<target host="m.refunite.org" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Refunite.org.xml]: static.refunite.org

File: src/chrome/content/rules/Refunite.org.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<!--
+	Nonfunctional hosts in *.refunite.org:
+
+	- www: mismatch
+-->
+<ruleset name="Refunite.org">
+	<target host="refunite.org" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Refunite.org.xml]: you can add `www.refunite.org` as a target with the rule `<rule from="http://www\.refunite\.org/" to="https://refunite.org/" />`

Comment (Bisaloo) [reply src/chrome/content/rules/Refunite.org.xml]: My bad, sorry, I made a typo:

<rule from="^http://www\.refunite\.org/" to="https://refunite.org/" />

Comment (jtracey) [reply src/chrome/content/rules/Refunite.org.xml]: ah, almost added that myself, but thought I must be misreading it somehow ;)
Thanks!

**Comment by Bisaloo** [comment]
Thank you!

**Pull request merged**